server/api/db/sqldb/db.py

```diff
@@ -971,6 +993,13 @@ def tag_artifacts(
         # this will also release the locks on the artifacts' rows
         self._commit(session, objects)
 
+        logger.debug(
+            "Locking artifacts in before tagging artifacts",
+            project=project,
+            tag=tag_name,
+            artifacts_keys=artifacts_keys,
+        )
```

Comment:
Same.

Reply:
same as above, but also fixed the log message